src/main/java/org/amahi/anywhere/service/AudioService.java

```diff
@@ -409,10 +409,6 @@ public void onAudioFocusChange(int audioFocus) {
 	private void handleAudioFocusGain() {
 		if (isAudioPlaying()) {
 			setUpAudioVolume();
-		} else {
-			if (audioFocus == AudioFocus.LOSS) {
-				playAudio();
```

Comment:
Removing this will break the functionality. Suppose a song was playing and you receive a call, now due to focus loss the song will be paused. But as soon as the call ends, the song must resume playing. This will not happen if we remove this line.